chirpy/api.py

```diff
@@ -181,10 +187,15 @@ def __call__(self, **kwargs):
         headers = {'Accept-Encoding': 'gzip'}
         headers.update(self.headers)
 
-        if method == 'GET':
+        if json_body is not None:
+            headers['Content-Type'] = 'application/json'
```

Comment:
I think you should indicate charset=UTF-8 encoding (`dumps()` defaults to utf8) here too just to be safe. I have no idea what the default is on the server side.